composer.json

```diff
@@ -21,7 +21,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^5.7|^7.0",
-        "friendsofphp/php-cs-fixer": "^2.9"
+        "friendsofphp/php-cs-fixer": "^2.9",
+        "ext-json": "*"
```

Comment:
any reason for this?

Reply:
`ext-json` is implicit dependency for now so i think it would be useful to make it explicit
BTW i found 2 more tests that fail 

- `UpdateCommandTest::it_updates_hooks_correctly_in_a_git_worktree`
- `AddCommandTest::it_adds_hooks_correctly_in_a_git_worktree`

looks line heisenbug in tests imho

Reply:
they were broken and fixed before. should be fine now

Reply:
the keyword is **heisenbug** ) fails on my local machine, but ok

Reply:
Hmm that's a bit strange. Do they always fail? what version of git do you have locally?

Reply:
2.1.4

Reply:
Maybe something related to that? Seems like a really old version because I have 2.22.0

Reply:
maybe. but still it's kinda bug

Reply:
Yeah. I'll try to take a look at it when I have some time. If you're able to investigate, just let me know